etc/scripts/utils_dejacode.py

```diff
@@ -15,7 +15,7 @@
 import requests
 import saneyaml
 
-from packaging import version as packaging_version
+from packvers import version as packaging_version
```

Comment:
etc/scripts comes from the https://github.com/nexB/skeleton and should be modified instead there

Reply:
Modification done there , please merge that PR https://github.com/nexB/skeleton/pull/71 :")